tasks/task023_cosmosqa_question_generation.json

```diff
@@ -7,7 +7,7 @@
         "cosmosqa"
     ],
     "Categories": [
-        "Question Generation"
+        "Contextual Question Generation"
```

Comment:
One might say that this also involves `Reasoning -> Commonsense Reasoning` according to the instructions.